doc/release/release_2.6.rst

```diff
@@ -501,7 +501,7 @@ Merged PRs
 - return earlier from `clique.graph_clique_number` (#4622)
 - More for projects page: TSP and Graph Isomorphism (#4620)
 - add recommended venv directory to .gitignore (#4619)
-- adding weight description to centrality metrices (#4610)
+- adding weight description to centrality metrics (#4610)
```

Comment:
I understand this typo exists in the PR title. Happy to drop, although PR titles are malleable...